### DISCUSSION TITLE
Problem in Discretization of initial texture file

### DISCUSSION THREAD

**Discussion by amanscnu** [mtex-toolbox/mtex#1074] [Ask Anything]
Hello Sir,

I tried to do the discretization of initial as-received texture file with 2000, 3000, 4000, and 5000 orientations. But after doing this with so many orientation, I am not getting change in volume fraction in texture component. 

![image](https://user-images.githubusercontent.com/85339580/120773379-40d07b00-c55c-11eb-9dac-0b4aec00bbbc.png)

**Comment by kilir** [comment]
Hi,
from my point of view, you may have to explain what you did and what you want to do.
Cheers,
Rüdiger

**Reply by amanscnu** [reply]
I have used this code to discretized the initial Cu samples orientations in order to further run for VPSC simulation
![image](https://user-images.githubusercontent.com/85339580/120795145-14285d80-c574-11eb-9948-075061cc4876.png)

But after discretization I am getting very low volume fraction of texture component than that of experimental volume fraction of texture component. Even when I increased my number of grains to be discretized from 2000 to 5000, then also there was very low volume fraction of texture components. 

I have a question, why discretization from MTEX are not giving nearby values of texture components (experimental volume fraction). Why the intensity/texture of discretized conditions shows nearly to half of the experimental. If there any error in my process. 

I want that texture component volume fraction of both discretized and experimental results shall be nearby

**Reply by kilir** [reply]
Hi,
the code you show does not really help, just shows that you use discreteSample and write Euler angles to a file.

if I understand correctly:
a) you have an odf and you determine the volumes of certain texture components. In Mtex? And how?
b) you sample from that ODF and you determine again an ODF? in Mtex? How?

Otherwise it remains very unclear what you do and it would be very much guessing what goes wrong.

Cheers,
Rüdiger

**Comment by amanscnu** [comment]
Hello,

a) I have used Mtex for ODF generation and volume fraction calculation of texture component: 
![image](https://user-images.githubusercontent.com/85339580/120948589-8455ff80-c77d-11eb-9659-d45107edc209.png)


b) I have a generated ODF from pole figure data and then I used discretized code for discretization of experimental ODF: I have used mtex code: 
![image](https://user-images.githubusercontent.com/85339580/120948729-d5fe8a00-c77d-11eb-8201-2be71e6c7f22.png)

![image](https://user-images.githubusercontent.com/85339580/120948831-09d9af80-c77e-11eb-82a4-e8f836e4d949.png)

![image](https://user-images.githubusercontent.com/85339580/120948856-18c06200-c77e-11eb-8aba-a98e8840956f.png)



I think you will get some idea about the process.

**Reply by ralfHielscher** [reply]
You should try harder to present your question in a condensed but exact way.
For your answer I understood:
a) you have a model ODF called `P_odf` 
b) you compute an ODF called `odf` from pole figure data
c) the volume percentage of `P_odf` and `odf` do not agree. 

May conclusion: The two ODFs `P_odf` and `odf` are different.
From your information there is no reason why they should not be different.

Ralf.
